tests/readme.md

```diff
@@ -0,0 +1,13 @@
+# Test readme
+
+## SMTP
+
+For testing SMTP you need [Mailpit](https://mailpit.axllent.org/) running on localhost.
+
+Since smtp_util uses STARTTLS you need to configure Mailpit for this. [More info here.](https://mailpit.axllent.org/docs/configuration/smtp/#smtp-with-starttls)
```

Comment:
If possible (and I know it is), please tell which terminal commands to run to do this. Your future self will thank you!